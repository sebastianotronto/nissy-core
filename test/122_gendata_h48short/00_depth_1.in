73
157
1

For longer test:

20000003
20000023
8

Short h48: generating depth 0
found 1
Short h48: generating depth 1
found 1
Short h48: generating depth 2
found 4
Short h48: generating depth 3
found 34
Short h48: generating depth 4
found 333
Short h48: generating depth 5
found 3815
Short h48: generating depth 6
found 45382
Short h48: generating depth 7
found 548562
Short h48: generating depth 8
found 6839723
