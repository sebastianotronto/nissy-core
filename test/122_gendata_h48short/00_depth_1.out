2
0 0
71075840 1
